circleci/github-release

```diff
@@ -28,6 +28,12 @@ tar jxf /tmp/github-release.tar.bz2 -C /tmp/ && sudo mv /tmp/bin/linux/amd64/git
 
 echo "Publishing github-release"
 TAG=$(head -n 1 VERSION)
+if [[ $TAG == "" && -e swagger.yml ]]; then  # fall back on swagger.yml version
+  TAG=$(grep -E 'version: \d+\.\d+\.\d+' swagger.yml  | cut -d: -f2 | sed 's/ //g')
+fi
```

Comment:
I worry about adding this fallback because it means that two files can define the version and it's not obvious which one would take precedence.  Also the adhoc regex parsing feels a bit fragile.

Ideas:
- Add VERSION files to wag repos
- Throw an error when both a VERSION and swagger.yml file are found.
- Eventually migrate this bash to a go binary where we can do proper yaml parsing.